frontend/packages/ux-editor/src/components/config/expressions/DynamicContent.tsx

```diff
@@ -18,19 +20,31 @@ import classes from './DynamicContent.module.css';
 interface ExpressionProps {
   component: FormComponent | FormContainer;
   dynamic: Dynamic;
-  onGetProperties: (dynamic: Dynamic) => {availableProperties: string[], expressionProperties: string[]}; // actions?
+  onGetProperties: (dynamic: Dynamic) => { availableProperties: string[], expressionProperties: string[] }; // actions?
   showRemoveDynamicButton: boolean;
   onAddDynamic: () => void;
+  successfullyAddedDynamicId: string;
   onRemoveDynamic: (dynamic: Dynamic) => void;
   onEditDynamic: (dynamic: Dynamic) => void;
 }
 
-export const DynamicContent = ({ component, dynamic, onGetProperties, onAddDynamic, showRemoveDynamicButton, onRemoveDynamic, onEditDynamic }: ExpressionProps) => {
+export const DynamicContent = ({
+                                 component,
+                                 dynamic,
+                                 onGetProperties,
+                                 showRemoveDynamicButton,
+                                 onAddDynamic,
+                                 successfullyAddedDynamicId,
+                                 onRemoveDynamic,
+                                 onEditDynamic
+                               }: ExpressionProps) => {
   const [selectedAction, setSelectedAction] = React.useState<string>(dynamic.property || 'default');
-  const [expressionElements, setExpressionElements] = React.useState<ExpressionElement[]>([...dynamic.expressionElements]); // default state should be already existing expressions
-  const { t } = useTranslation();
+  const [expressionElements, setExpressionElements] = React.useState<ExpressionElement[]>(dynamic.expressionElements && [...dynamic.expressionElements] || []); // default state should be already existing expressions
```

Comment:
What do you think of this syntax instead? :)
```suggestion
  const [expressionElements, setExpressionElements] = React.useState<ExpressionElement[]>(dynamic.expressionElements ? [...dynamic.expressionElements] : []); // default state should be already existing expressions
```